joint_state_broadcaster/include/joint_state_broadcaster/joint_state_broadcaster.hpp

```diff
@@ -100,14 +101,16 @@ class JointStateBroadcaster : public controller_interface::ControllerInterface
   //  we store the name of joints with compatible interfaces
   std::vector<std::string> joint_names_;
   std::shared_ptr<rclcpp::Publisher<sensor_msgs::msg::JointState>> joint_state_publisher_;
-  sensor_msgs::msg::JointState joint_state_msg_;
+  std::shared_ptr<realtime_tools::RealtimePublisher<sensor_msgs::msg::JointState>>
+    realtime_joint_state_publisher_ = nullptr;
```

Comment:
It is really needed to set the initial value to `nullptr`?

If so, we should add this to all controllers (`good-first-issue`). If needed please open follow-up issue.

Reply:
```suggestion
    realtime_joint_state_publisher_;
```
du has recht, not strictly needed